src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -11,13 +11,20 @@ export interface WithAnnotationsProps {
     onPreviewDestroy: () => void;
 }
 
+type ActiveChangeEvent = {
+    annotationId: string | null;
+    fileVersionId: string;
```

Comment:
Is this optional/nullable or do we expect it to be defined even if `annotationId` is null? Seems a little odd.

Reply:
I was expecting `fileVersionId` to always be defined since null is actually a valid active annotation value